gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoints-group/configuration/api-endpoints-group-configuration.component.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Subject } from 'rxjs';
+import { map, takeUntil } from 'rxjs/operators';
+
+import { ConnectorService } from '../../../../../../services-ngx/connector.service';
+
+@Component({
+  selector: 'api-endpoints-group-configuration',
+  template: require('./api-endpoints-group-configuration.component.html'),
+  styles: [require('./api-endpoints-group-configuration.component.scss')],
+})
+export class ApiEndpointsGroupConfigurationComponent implements OnInit, OnDestroy {
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
+
+  @Input() groupConfigurationControl: FormControl;
+
+  public schemaForm: unknown;
+
+  constructor(private readonly connectorService: ConnectorService) {}
```

Comment:
You should not use this service: `ConnectorService`. It's the one used for V2 API.
The schema-form returned by this service are taken from this kind of plugins:
 - https://github.com/gravitee-io/gravitee-connector-http
 - https://github.com/gravitee-io/gravitee-connector-kafka
 
For V4 APIs, you need the get the schemas from the "new" endpoints:
 - https://github.com/gravitee-io/gravitee-api-management/tree/master/gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-http-proxy
 - https://github.com/gravitee-io/gravitee-endpoint-kafka
 - ...

To get these schemas, you can use this typescript service: `ConnectorPluginsV2Service`